packages/saltcorn-data/package.json

```diff
@@ -71,6 +71,7 @@
     "node-fetch": "2.6.9",
     "pluralize": "^8.0.0",
     "process": "0.11.10",
+    "punnycode": "^2.3.1 ",
```

Comment:
should this be punycode (on n) ?

Reply:
Thanks yes, I wanted it to be punycode. The webpack build failed bundling this module.